packages/reports/addon/components/common-actions/get-api.js

```diff
@@ -34,7 +47,13 @@ export default class GetApiActionComponent extends Component {
     let request = this.request.serialize();
     return this.facts.getURL(request, { dataSourceName: request.dataSource });
```

Comment:
Just realized this will need to update https://github.com/yahoo/yavin/blob/4d0274a58f67ddb6269b866094452e5c3de3b4e8/packages/data/addon/services/navi-facts.ts#L57-L58

so that it queries the correct adapter